### PR TITLE
doc: update pagure commit flagging description

### PR DESCRIPTION
**Pull request by praiskup**

### CODE REVIEW

File: doc/user_documentation.rst [hunk 1]
```diff
@@ -296,13 +296,24 @@ of the names ``<coprname>:pr:<pr_id>``. ``<pr_id>`` is ID of the pull request op
 PR/commit flagging
 ^^^^^^^^^^^^^^^^^^
 
-If you would like to get your commits and pull requests in Pagure flagged with build results for each change, go to project settings in your Pagure project. Then:
+Since you have your automatic Pagure builds configured (instructions above), the
+remaining part is "how do I analyze the build results?".  Upon the
+``Pagure => Copr`` (build) request, we need the ``Copr => Pagure`` (build state)
```
Comment (FrostyX) [inline doc/user_documentation.rst:301]: > Upon the ``Pagure => Copr`` (build) request, we need the ``Copr => Pagure`` (build state)

This sentence is IMHO too hard to understand

**Comment by praiskup** [comment]
Thank you for the feedback!  Trying differently, can you perhaps propose a better wording?

File: doc/user_documentation.rst [hunk 2]
```diff
@@ -296,13 +296,24 @@ of the names ``<coprname>:pr:<pr_id>``. ``<pr_id>`` is ID of the pull request op
 PR/commit flagging
 ^^^^^^^^^^^^^^^^^^
 
-If you would like to get your commits and pull requests in Pagure flagged with build results for each change, go to project settings in your Pagure project. Then:
+Above you configured a Pagure instance that auto-triggers Copr builds (*request*).
+How to analyze the corresponding build results from the Pagure web UI?  You can
+configure Copr to notify Pagure instance (*response*) about the build results
+using the "commit flagging" feature:
```
Comment (FrostyX) [inline doc/user_documentation.rst:302]: Thank you for the update, makes more sense to me now.

Nit: I am still confused by the (*request*) and (*response*) brackets but the text works without them, so I would probably suggest just removing them. Possibly adding a new sentence at the end of the paragraph, saying what they used to say.

But +1 even to the current state.

**Review by FrostyX — APPROVED** [review]

**Review by nikromen — APPROVED** [review]
lgtm

can we merge #2662 earlier please?

**Comment by praiskup** [comment]
2662 merged, I rebased this one and added one more commit moving the pagure integration part into a separate page.

**Comment by FrostyX** [comment]
+1

**Pull request merged**